exams/models.py

```diff
@@ -35,3 +38,61 @@ class ExamProfile(Model):
 
     def __str__(self):
         return 'Exam Profile "{0}" with status "{1}"'.format(self.id, self.status)
+
+
+class ExamAuthorization(TimestampedModel):
+    """
+    Tracks state of an exam authorization
```

Comment:
It's the record we need to send to Pearson authorizing an individual for a specific exam. This model exists so we can track state on our side.